spec/consensus/light-client.md

```diff
@@ -316,3 +316,27 @@ func Backwards(h1,h2) bool {
   return (hash(h2) == old.Header.hash)
  }
 ```
+
+### Failure Detection Module
+
+The light client must be able to detect failures in the trust assumptions.
+Ideally, it should never trust a header, which was forged by malicious
+validators.
+
+To meet the above requirement, it **must be connected to at least one honest
+full node**.
```

Comment:
I agree. This is necessary. In case of no fork on the main chain this is also sufficient. 
If there is a fork on the main chain, this might not be sufficient anymore: a fork on the main chain means that two full nodes have decided on two different headers for the same height. Then the requirement would at least require that the lite client is connected to one honest full node on each branch of the fork.